src/main/scala/com/horizon/exchangeapi/PatternsRoutes.scala

```diff
@@ -836,6 +861,17 @@ trait PatternsRoutes extends JacksonSupport with AuthenticationSupport {
       ]
     }
   ],
+  "secretBinding": [
+     {
+       "serviceOrgid": "string",
+        "serviceUrl": "string",
+        "serviceArch": "amd64",
+        "serviceVersionRange": "x.y.z",
+         "secrets": [
```

Comment:
Make sure `"secrets": ...` is structurally similar to `"inputs": ...` above, but with the context of storing secrets instead of inputs.